src/site/content/en/blog/economic-times-cwv/index.md

```diff
@@ -0,0 +1,163 @@
+
+In addition to switching to a faster DNS provider and optimising images here are some of the techniques we applied we covered to improve LCP.
+
+### Critical requests first
+
+As all modern browsers limit the concurrent number of requests, developers need to prioritize loading the critical content first. To load a complex web page we need to download assets such as header elements, CSS, JS resources, hero image, article body, comments, other related news, footer, and ads. We evaluated what elements were required for LCP, and provided the preference to load those items first to improve LCP. We also deferred the calls that were not part of the initial page rendering.
+
+### Text appearance
+
+We experimented with the [`font-display` property](https://developer.mozilla.org/docs/Web/CSS/@font-face/font-display) as this impacts both LCP and CLS. We tried `font-display: auto;` initially and then switched over to `font-display: swap;`. This would render the text initially in the best matching and available font and then switch over to the font which had been downloaded. This resulted in our text rendering quickly, independent of network speed.
+
+### Better Compression
+
+[Brotli](http://github.com/google/brotli/) is an alternative compression algorithm to Gzip and Deflate developed by Google. We replaced our fonts and assets and changed server compression from Gzip to Brotli to achieve a smaller footprint:
+
+- Javascript files are 15% smaller than gzip.
+- HTML files are 18% smaller than gzip.
+- CSS and font files are 17% smaller than gzip.
+
+### Preconnect to third-party domains
+
+[`preconnect`](/preconnect-and-dns-prefetch/#establish-early-connections-with-relpreconnect) should be used carefully as it can still take up valuable CPU time, and delay other important resources, especially on secure connections.
+
+However, if it's known that a fetch for a resource on a third-party domain will occur, `preconnect` is good. If it only happens occasionally on a high traffic website, `preconnect` might trigger unnecessary TCP and TLS work. Thus [`dns-prefetch`](/preconnect-and-dns-prefetch/#resolve-domain-name-early-with-reldns-prefetch) was a better fit for third-party resources&mdash;for example, social media, analytics, etc&mdash;to perform DNS lookups ahead of time.
+
+### Break up code into chunks
+
+In the head section we only loaded those resources which contain either an essential part of the business logic, or were [critical](https://github.com/addyosmani/critical) for above the fold page rendering. Furthermore, we split our code into chunks with [code splitting](https://loadable-components.com/docs/code-splitting/). This helped us to further improve page LCP.
+
+### Better caching
+
+For all the frontend routes, we added a Redis layer which served the templates from a cache through a specific time period. This reduces the computation time on the server and builds the whole UI code in each request, thus decreasing LCP in subsequent requests.
```

Comment:
A couple of things.

* I had to look up Redis. Please provide a link.
* I don't know what you mean by 'through a specific time period'.